azure-kusto-data/azure/kusto/data/kcsb.py

```diff
@@ -128,12 +128,13 @@ def __init__(self, connection_string: str):
         for kvp_string in connection_string.split(";"):
             key, _, value = kvp_string.partition("=")
             keyword = self.ValidKeywords.parse(key)
+            value_stripped = value.strip()
             if keyword.is_str_type():
-                self[keyword] = value.strip()
```

Comment:
You probably still want to strip, in addition to removing the trailing slash.

Reply:
See line 131